signer/apk/apk.go

```diff
@@ -91,7 +105,7 @@ func (s *APKSigner) SignFile(input []byte, options interface{}) (signer.SignedFi
 	if err != nil {
 		return nil, errors.Wrap(err, "apk: failed to sign APK")
 	}
-	signedFile, err = repackAndAlignJAR(input, manifest, sigfile, p7sig)
+	signedFile, err = repackAndAlignJAR(input, manifest, sigfile, p7sig, opt.ZIP)
```

Comment:
should that be renamd to `repack and maybe align jar` ? :trollface:

Reply:
Yeah, I'm planning to follow up with a PR to fix alignment that splits it repack and align functions like the XPI jar.go.